### PR TITLE
"Apex Debug: Configure Exceptions" command is visible in non-sfdx projects

### PR DESCRIPTION
**Pull request by cademay**
### What does this PR do?

fixed visibility bug; added prefix to palette command text

### What issues does this PR fix or reference?
@W-6197646@

### CODE REVIEW

File: packages/salesforcedx-vscode-apex-debugger/package.nls.json [hunk 1]
```diff
@@ -9,6 +9,6 @@
   "trace_text": "Produce diagnostic output. You can set this to true, false, or one or more selectors separated with commas. Selectors include 'all' (very detailed output; the equivalent of true), 'protocol', 'variables', 'launch', 'breakpoints' and 'streaming'.",
   "configuration_title": "Salesforce Apex Debugger Configuration",
   "connection_timeout_ms_description": "Connection timeout for Apex Debugger API requests (in milliseconds).",
-  "exception_breakpoint_command_text": "Apex Debug: Configure Exceptions",
+  "exception_breakpoint_command_text": "SFDX: Configure Apex Debug Exceptions",
```
Comment (cademay) [inline packages/salesforcedx-vscode-apex-debugger/package.nls.json]: @hbc-bindu does this message look okay?

thank you

Comment (hbc-bindu) [reply packages/salesforcedx-vscode-apex-debugger/package.nls.json]: Hi @cademay, looks good to me. Thanks.

**Comment by lcampos** [comment]
This PR also needs to update our docs for Interactive Debugger (https://forcedotcom.github.io/salesforcedx-vscode/articles/apex/interactive-debugger)

File: docs/articles-jp/apex/interactive-debugger.md [hunk 2]
```diff
@@ -69,9 +69,9 @@ VS Code で初めて Apex デバッガを使用するときは、次の設定手
 
 デバッグセッション中に例外が発生した場合に、Apex デバッガの実行を停止するには、例外にブレークポイントを設定します。例外ブレークポイントに達すると、例外の原因となったコード行でデバッガが一時停止します。[Debug \(デバッグ\)] ビューの [Call Stack \(コールスタック\)] パネルに、例外の名前が表示されます。
 
-例外ブレークポイントを設定するには、Ctrl+Shift+P キー \(Windows、Linux\) または Cmd+Shift+P キー \(macOS\) を押してコマンドパレットを開き、**[Apex Debug: Configure Exceptions \(Apex デバッグ: 例外を設定\)]** を選択します。使用可能な例外のリストに、[exceptions in the `System` namespace](https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/apex_classes_exception_methods.htm) と、`Exception` を拡張するプロジェクトの Apex クラスが表示されます。リストから例外を選択して、**[Always break \(常に中断\)]** をクリックします。
```
Comment (cademay) [inline docs/articles-jp/apex/interactive-debugger.md]: @ntotten We noticed that we have some Japanese text going on in these files. In updating the label text for this palette feature, we are wondering how to go about making sure the Japanese text gets updated properly as well. At your convenience, please let us know your thoughts on this

Thank you,
Cade

**Pull request merged**